PushNotification/PushNotification/PushNotification.Plugin.Android/PushNotificationsReceiver.cs

```diff
@@ -1,22 +1,7 @@
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using Android.Support.V4.Content;
 using Android.App;
 using Android.Content;
-using Android.OS;
-using Android.Runtime;
-using Android.Views;
-using Android.Widget;
 using Android.Gms.Gcm;
 
-[assembly: Permission(Name = "@PACKAGE_NAME@.permission.C2D_MESSAGE")]
```

Comment:
What is the advange of having this on the assembly?

Already worked automatically by been here.